setenv.sh

```diff
@@ -1,14 +1,44 @@
 # if running the kdb+tick example, change these to full paths
 # some of the kdb+tick processes will change directory, and these will no longer be valid
 export TORQHOME=${PWD}
-export KDBCONFIG=${TORQHOME}/config
-export KDBCODE=${TORQHOME}/code
-export KDBLOG=${TORQHOME}/logs
-export KDBHTML=${TORQHOME}/html
-export KDBLIB=${TORQHOME}/lib
+export TORQDATA=${PWD}
+export TORQAPPHOME=${PWD}
+export KDBLOG=${TORQDATA}/logs
+export KDBHTML=${TORQDATA}/html
+export KDBLIB=${TORQDATA}/lib
+export KDBAPPCODE=${TORQAPPHOME}/code
+export KDBCONFIG=${TORQAPPHOME}/config
+export KDBCODE=${TORQAPPHOME}/code
+
+# sets the application specific configuration directory
+export KDBAPPCONFIG=${TORQHOME}/appconfig
+# set KDBBASEPORT to the default value for a TorQ Installation
+export KDBBASEPORT=41000
+# set DEFAULTCSV to the default process csv 
```

Comment:
inline comments seem like they would be more consistent with the style of `torq.sh`